### ISSUE TITLE
Could there be a way to get the ip camera feed?

### ISSUE THREAD

**Issue by pelebel** [XaF/qolsysgw#158]
So I cut the cord with alarm.com, just wondering if it would be a possibility.

Great job integrating the panel, and many thanks!

**Comment by XaF** [comment]
I don't believe this information is provided from the panel control4 feed - I'd be totally happy to integrate it if the data is available, but it does not seem it is!
